README.md

```diff
@@ -15,10 +15,13 @@ For example it is often used to slightly massage data before using it with Totar
 
 ## Branches
 
-| Moodle verion     | Branch  | PHP        |
-| ----------------- | ------- | ---------- |
-| Moodle 2.9 to 3.8 | master  | 5.6 - 7.0+ |
-| Totara 12+        | master  | 5.6 - 7.0+ |
+| Moodle verion      | Branch  | PHP        |
+| ------------------ | ------- | ---------- |
+| Moodle 3.3 to 3.8  | master  | 7.1 - 7.2+ |
+| Moodle 3.9 to 3.10 | master  | 7.2 - 7.4+ |
+| Totara 12+         | master  | 5.6 - 7.0+ |
+
+NOTE: Moodle 3.11 is not supported. Multiple unit tests are failing.
```

Comment:
I would avoid saying that. There might be issue that need to be solved - yes. Let's remove this line, add Moodle 3.11 and master to github actions and create an issue for failing unit tests